### ISSUE TITLE
[Machine Learning] Multi-Class Image Classifier using Deep AutoViML

### ISSUE THREAD

**Issue by simonndiritu477** [section-engineering-education/engineering-education#6231]
#### Proposed title of article
[Machine Learning] Multi-Class Image Classifier using Deep AutoViML

#### Proposed article introduction
Image classification is one of the supervised machine learning problems which aims to categorize the images of a dataset into their respective categories or labels. Multi-Class image classification is a task in computer vision, where we categorize an image into three or more classes.  Deep Autoviml is an AutoML library for building deep learning models using TensorFlow and Keras.

Deep AutoViML will automatically load a wide variety of performant DNN architectures such as deep and wide, deep and cross models. The loaded model will then be used for image classification. Deep AutoViML will also automatically select the best model, add pre-processing layers for feature transformation and do selective feature engineering.

In this tutorial, we will use Deep AutoViML to build a custom model that can identify images of hands playing rock-paper-scissors games. Our image will be made up of three classes.

#### Key takeaways 
1. What is multi-class image classification.
2. Installing Deep AutoViML.
3. Image dataset pre-processing.
4. Using Deep AutoViML to automatically select the best model.
5. Fine-tuning the model to give the best prediction results.
#### Article quality
This tutorial is unique because we will compare multi-label and multi-class image classification. This will give the reader a deeper understanding of multi-class image classification and how to differentiate the two classification types. The tutorial will also explain in details the techniques that AutoViMl uses to select the best model for image selection. We will explore how  AutoViML searches and fine-tunes the neural network layers to get the best model architecture. By the end, we will have a fully optimized neural network that will give the best results.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by ahmadmardeni1** [comment]
Sounds like a helpful topic - let's please be sure it adds value beyond what is in any official docs and/or what is covered in other blog sites. (the articles should go beyond a basic explanation - and it is always best to reference any EngEd article and build upon it). @simonndiritu477 

Please be attentive to grammar/readability and make sure that you put your article through a thorough editing review prior to submitting it for final approval. (There are some great free tools that we reference in EngEd resources.)
ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.

Please be sure to double-check that it does not overlap with any existing EngEd articles, articles on other blog sites, or any incoming EngEd topic suggestions (if you haven't already) to avoid any potential article closure, please reference any relevant EngEd articles in yours. - Approved
